src/Symfony/Component/AssetMapper/Tests/Compiler/JavaScriptImportPathCompilerTest.php

```diff
@@ -641,4 +641,23 @@ public function testErrorMessageAvoidsCircularException()
         // should not be caught.
         $this->assertSame($content, $compiled);
     }
+
+    public function testCompilerThrowsExceptionOnPcreError()
+    {
+        $compiler = new JavaScriptImportPathCompiler($this->createMock(ImportMapConfigReader::class));
+        $content = str_repeat('foo "import *  ', 50);
+        $javascriptAsset = new MappedAsset('app.js', '/project/assets/app.js', publicPathWithoutDigest: '/assets/app.js');
+        $assetMapper = $this->createMock(AssetMapperInterface::class);
+
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage('Failed to compile JavaScript import paths in "/project/assets/app.js". Error: "Backtrack limit exhausted".');
```

Comment:
Careful: Those error messages have been changed in the latest PCRE2 release. 🙈

Reply:
Arf :( How would you handle this ?

Reply:
I've just tested locally with PCRE2 10.43. Your test still passes, so I guess you've picked an error where die message didn't change. Let's 🤞🏻 that it stays that way. 😅